docs/sources/mimir/references/architecture/components/query-scheduler/index.md

```diff
@@ -27,17 +27,13 @@ The following flow describes how a query moves through a Grafana Mimir cluster:
 
 ## Benefits of using the query-scheduler
 
-Query-scheduler enables the scaling of query-frontends. You might experience challenges when you scale query-frontend. To learn more about query-frontend scalability limits, refer to [Why query-frontend scalability is limited](../query-frontend/#why-query-frontend-scalability-is-limited).
-
-### How query-scheduler solves query-frontend scalability limits
-
-When you use the query-scheduler, the queue is moved from the query-frontend to the query-scheduler, and the query-frontend can be scaled to any number of replicas.
-
-The query-scheduler is affected by the same scalability limits as the query-frontend, but because a query-scheduler replica can handle high amounts of query throughput, scaling the query-scheduler to a number of replicas greater than `-querier.max-concurrent` is typically not required, even for very large Grafana Mimir clusters.
+Query-scheduler enables the scaling of query-frontends by moving queuing of requests to a separate component.
+The query-scheduler prevents queries that only require ingesters from being affected by degradation of store-gateways and vice versa.
+The query-scheduler ensures tenant fairness using a simple round-robin between all tenants with non-empty query request queues.
```

Comment:
Minor: could potentially go a little less technical-sounding here, instead of 

> between all tenants with non-empty query request queues.

maybe

> between all tenants with query requests enqueued

or

> between all tenants with active queries

or some mashup of the above